.github/workflows/check-links-in-prod.yaml

```diff
@@ -12,15 +12,54 @@ jobs:
       - name: Pull images
         run: docker pull ghcr.io/linkchecker/linkchecker:latest
 
-      - name: Check links
+      - name: Check links in overview pages
         run: |
           docker run --rm --name linkchecker \
             ghcr.io/linkchecker/linkchecker:latest \
-              https://docs.giantswarm.io/ \
-              --check-extern \
-              -t 1 \
-              --ignore-url="^https://github.com/giantswarm/docs/.*" \
-              --ignore-url="^https://.*example\.com/.*" \
-              --ignore-url="^https://my-org\.github\.com/.*" \
-              --ignore-url="^https://github\.com/giantswarm/giantswarm/.*" \
-              --ignore-url=".*gigantic\.io.*"
+            https://docs.giantswarm.io/overview \
+            --check-extern \
+            -t 1
+
+      - name: Check links in getting-started pages
+        run: |
+          docker run --rm --name linkchecker \
+            ghcr.io/linkchecker/linkchecker:latest \
+            https://docs.giantswarm.io/getting-started \
+            --check-extern \
+            -t 1
+
+      - name: Check links in tutorials pages
+        run: |
+          docker run --rm --name linkchecker \
+            ghcr.io/linkchecker/linkchecker:latest \
+            https://docs.giantswarm.io/tutorials \
+            --check-extern \
+            -t 1
+
+      - name: Check links in reference pages
+        run: |
+          docker run --rm --name linkchecker \
+            ghcr.io/linkchecker/linkchecker:latest \
+            https://docs.giantswarm.io/reference \
+            --check-extern \
+            -t 1
+
+      - name: Check links in support pages
+        run: |
+          docker run --rm --name linkchecker \
+            ghcr.io/linkchecker/linkchecker:latest \
+            https://docs.giantswarm.io/support \
+            --check-extern \
+            -t 1
+
+      - name: Check links in changelogs
+        run: |
+          docker run --rm --name linkchecker \
+            ghcr.io/linkchecker/linkchecker:latest \
+            https://docs.giantswarm.io/changes \
+            -t 1 -r 2 \
+            --ignore-url="^https://github.com/giantswarm/docs/.*" \
+            --ignore-url="^https://.*example\.com/.*" \
+            --ignore-url="^https://my-org\.github\.com/.*" \
+            --ignore-url="^https://github\.com/giantswarm/giantswarm/.*" \
+            --ignore-url=".*gigantic\.io.*"
```

Comment:
This makes it easier to consume the output and does not fall into the whole of deep validation (we do have not control over when the old changelog points to a non-existing URL)